components/expenses/Transaction.js

```diff
@@ -133,13 +133,7 @@ class Transaction extends React.Component {
     }
 
     const amountToDisplay = ['ORGANIZATION', 'USER'].includes(collective.type) ? netAmountInCollectiveCurrency : amount;
-    let precision = 0;
-
-    // Check if the remainder is not 0, this check is important in order to know the amount that needs precision
-    // i.e display $10.21 as $10.21 instead of $10
-    if ((amountToDisplay / 100) % 1 !== 0) {
-      precision = 2;
-    }
+    let precision = 2;
```

Comment:
If the variable is not modified, we shoud use `const`. But I'm not even sure we need a variable for this as we could set the value directly in the props